pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -1310,4 +1312,17 @@ private void parseFullyQualifiedFunctionName(String fqfn, FunctionConfig functio
         }
     }
 
+    @Parameters(commandDescription = "Get the list of Pulsar Functions supported by Pulsar cluster")
+    public class ListBuiltInFunctions extends BaseCommand {
+        @Override
+        void runCmd() throws Exception {
+            getAdmin().functions().getBuiltInFunctions()
+                    .forEach(function -> {
+                        System.out.println(function.getName());
```

Comment:
could we create a common method to display this kind of stuff ? (sinks, sources, functions..)

Reply:
Between source and sink maybe. But between Function and Connectors, we handle distinct classes that are unrelated (FunctionDefinition and ConnectorDefinition) so I don't think we can factorize.